life_expectancy/tests/conftest.py

```diff
@@ -6,11 +6,11 @@
 
 
 @pytest.fixture(autouse=True)
-def run_before_and_after_tests() -> None:
```

Comment:
Porque tiraste o data type? Mais por curiosidade do que outra coisa

Reply:
olá André,

o mypy dá erro pois o type esperado é um "Generator", contudo nessa função não vamos devolver nada pois serve apenas como está  para realizar os testes e fazer teardown do file_path. para evitar o erro do mypy retirei o type.

copio-te o erro do mypy que me dá no meu pre-commit:

life_expectancy/tests/conftest.py:9: error: The return type of a generator function should be "Generator" or one of its supertypes  [misc]
Found 1 error in 1 file (checked 1 source file)

Reply:
mypy can get sometimes confused with decorated functions.

Other possible solutions:
- Ignore the specific mypy error, by adding: # type: ignore[<code>]
- Use `typing.no_type_check` [decorator](https://docs.python.org/3/library/typing.html#typing.no_type_check) (but I don't have much experience with it)

Reply:
thank you @MrCordeiro !